app/assets/javascripts/application.js

```diff
@@ -13,3 +13,6 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+var foo = 1;
+"double quotes so hound might yell"
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.